api/src/main/java/org/pmiops/workbench/access/AccessTierServiceImpl.java

```diff
@@ -118,7 +107,7 @@ public void removeUserFromRegisteredTier(DbUser user) {
    * @param user the DbUser in the user-accessTier mapping we're updating
    * @param accessTier the DbAccessTier in the user-accessTier mapping we're updating
    */
-  private void addUserToTier(DbUser user, DbAccessTier accessTier) {
+  public void addUserToTier(DbUser user, DbAccessTier accessTier) {
```

Comment:
Before this PR: these methods do the work of `addUserToRegisteredTier()`
In this PR: now they are also callable independently 
After this PR: remove `addUserToRegisteredTier()` so these methods are the primary interface